public/app/features/logs/components/panel/LogListControls.tsx

```diff
@@ -735,7 +735,7 @@ const WrapLogMessageButton = ({ expanded }: LogSelectOptionProps) => {
       tooltip={tooltip}
       label={wrapStateText}
       buttonAriaLabel={tooltip}
-      customTagText={'+'}
+      customTagText={prettifyJSON ? '+' : ''}
```

Comment:
Small regression with the expandable menu. This has 3 states:

- Disabled
- Enabled
<img width="33" height="38" alt="imagen" src="https://github.com/user-attachments/assets/9ee9ba49-531f-4914-b602-717e391b1dfc" />
- Enabled with prettify JSON
<img width="38" height="37" alt="imagen" src="https://github.com/user-attachments/assets/535ede28-a1ef-47de-9d57-f5480d960eb8" />

This was also added to the WIP docs.